src/Component.php

```diff
@@ -204,8 +206,10 @@ private function handleFor( DOMNode $node, array $data ) {
 		if ( $node->hasAttribute( 'v-for' ) ) {
 			list( $itemName, $listName ) = explode( ' in ', $node->getAttribute( 'v-for' ) );
 			$node->removeAttribute( 'v-for' );
+			$node->removeAttribute( ':key' );
+			$listNameExpression = new VariableAccess( explode( '.', $listName ) );
 
-			foreach ( $data[$listName] as $item ) {
+			foreach ( $listNameExpression->evaluate( $data ) as $item ) {
```

Comment:
AFAICT we can also use Peast here:

```suggestion
			foreach ( $this->app->evaluateExpression( $listName, $data ) as $item ) {
```

(And also remove the `$listNameExpression` assignment, but GitHub doesn’t seem to support a suggestion spanning multiple diff hunks.)

Maybe we could also replace the `explode( ' in ', $node->getAttribute( 'v-for' ) )` with Peast later.